docs/extend/plugins_volume.md

```diff
@@ -52,7 +52,8 @@ containers.
 **Request**:
 ```
 {
-    "Name": "volume_name"
+    "Name": "volume_name",
+    "Opts": {}
```

Comment:
(As in; mention that you can pass additional (driver specific) options to the driver)